packages/salesforce-adapter/src/elements_url_retreiver/lightining_url_resolvers.ts

```diff
@@ -66,8 +66,13 @@ const METADATA_TYPE_TO_URI: Record<string, string> = {
   FieldSet: 'FieldSets',
 }
 
-const getTypeIdentifier = async (element?: Element): Promise<string | undefined> =>
-  (element === undefined ? undefined : (getInternalId(element) ?? apiName(element)))
+const getTypeIdentifier = async (element?: Element): Promise<string | undefined> => {
+  if (element === undefined) {
+    return undefined
+  }
+  const internalId = getInternalId(element)
+  return internalId !== null && internalId !== undefined && internalId !== '' ? internalId : apiName(element)
```

Comment:
I think this is the same. But need to check every use case.
Also I think this should have a UT as well.
```suggestion
  return internalId ? internalId : apiName(element)
```

Reply:
This would convert into the original code using `??`, which doesn't apply to empty string

Reply:
But `!internalId` would cover all the cases, no?